airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -70,14 +70,48 @@ definitions:
         type: string
   AirbyteStateMessage:
     type: object
-    additionalProperties: true
-    required:
-      - data
+    additionalProperties: false
```

Comment:
Why are we changing this to `false`? I think that will make it harder in the future to add/remove fields to this object in a backwards-compatible way

Reply:
This field allows too many thing it is a `Map<String, Object>`, we can't know what is in it and we should avoid using it. I don't think it is being used at the moment and we shouldn't be using it in the future.